cli/query/query.go

```diff
@@ -0,0 +1,99 @@
+package query
+
+import (
+	"bytes"
+	"fmt"
+	"strconv"
+	"text/tabwriter"
+
+	"github.com/nspcc-dev/neo-go/cli/options"
+	"github.com/nspcc-dev/neo-go/pkg/core/transaction"
+	"github.com/nspcc-dev/neo-go/pkg/encoding/fixedn"
+	"github.com/nspcc-dev/neo-go/pkg/rpc/response/result"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+	"github.com/urfave/cli"
+)
+
+// NewCommands returns 'wallet' command.
+func NewCommands() []cli.Command {
+	queryTxFlags := append([]cli.Flag{
+		cli.BoolFlag{
+			Name:  "verbose, v",
+			Usage: "Output full tx info and execution logs",
+		},
+	}, options.RPC...)
+	return []cli.Command{{
+		Name:  "query",
+		Usage: "query",
+		Subcommands: []cli.Command{
+			{
+				Name:   "tx",
+				Usage:  "query tx status",
+				Action: queryTx,
+				Flags:  queryTxFlags,
+			},
+		},
+	}}
+}
+
+func queryTx(ctx *cli.Context) error {
+	args := ctx.Args()
+	if len(args) == 0 {
+		return cli.NewExitError("Transaction hash is missing", 1)
+	}
+
+	txHash, err := util.Uint256DecodeStringLE(args[0])
```

Comment:
Let's use `util.Uint256DecodeStringLE(strings.TrimPrefix(args[0], "0x"))`, because we can handle this case for uint160.